crates/env/Cargo.toml

```diff
@@ -61,22 +61,24 @@ ink = { path = "../ink" }
 [features]
 default = ["std"]
 std = [
-    "ink_allocator/std",
-    "ink_prelude/std",
-    "ink_primitives/std",
-    "ink_storage_traits/std",
-    "ink_engine/std",
-    "scale/std",
-    "scale-decode",
-    "scale-encode",
-    "scale-info/std",
-    "secp256k1",
-    "schnorrkel",
-    "num-traits/std",
+	"ink_allocator/std",
+	"ink_prelude/std",
+	"ink_primitives/std",
+	"ink_storage_traits/std",
+	"ink_engine/std",
+	"scale/std",
+	"scale-decode",
+	"scale-encode",
+	"scale-info/std",
+	"secp256k1",
+	"schnorrkel",
+	"num-traits/std",
```

Comment:
Please revert, seems like your IDE converted spaces to tabs.

Reply:
that's me using zepter to auto fix the missing std propagation, I think we should keep this, so we can eventually use this tool in CI to fix issues

Reply:
actually you are right the tool should be more flexible, will fix this up